webapi/Options/PluginOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace CopilotChat.WebApi.Options;
+
+/// <summary>
+/// Option for a single plugin.
+/// </summary>
+public class Plugin
+{
+    /// <summary>
+    /// The name of the plugin.
+    /// </summary>
+    public string Name { get; set; } = string.Empty;
+
+    /// <summary>
+    /// The url of the plugin.
+    /// </summary>
+    public Uri Url { get; set; } = new Uri("http://localhost");
```

Comment:
To keep definitions aligned, this should be named ManifestDomain since we're following the standard defined here: https://platform.openai.com/docs/plugins/production/domain-verification-and-security

On the front end, we also refer to this URL as the manifest domain through the wizard 

```suggestion
    public Uri ManifestDomain { get; set; } = new Uri("http://localhost");
```